widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/NameUtils.java

```diff
@@ -76,9 +76,13 @@ public static String reservedCharListForDisplay( ) {
     return getReservedCharsDisplay();
   }
 
-  public static native String URLEncode( String value )
+  public static String URLEncode( String value ) {
+    return URLEncode( value, true );
+  }
+
+  public static native String URLEncode( String value, Boolean doubleEncodeSlashes )
```

Comment:
New parameter is always being ignored.

Reply:
this would be okay, if allowing for future functionality. the author would just have to explain the design decision.